src/screens/Order/OrdersHelper.res

```diff
@@ -0,0 +1,73 @@
+open CommonAuthForm
+open CommonInputFields
+@react.component
+let make = (~input, ~options) => {
+  let (selectedOption, setSelectedOption) = React.useState(_ => "Select Amount")
+  let form = ReactFinalForm.useForm()
+  let onClick = _ => {
+    form.submit()->ignore
+  }
+  let input: ReactFinalForm.fieldRenderPropsInput = {
+    name: "amount",
+    onBlur: _ => (),
+    onChange: ev => {
+      let newValue = ev->Identity.formReactEventToString
+      if newValue != selectedOption && newValue != "" {
+        setSelectedOption(_ => newValue)
+        form.change("amount_filter.end_amount", JSON.Encode.null)
+        form.change("amount_filter.start_amount", JSON.Encode.null)
+      }
+    },
+    onFocus: _ => (),
+    value: {
+      selectedOption->JSON.Encode.string
+    },
+    checked: true,
+  }
+
+  let renderCommonFields = (field, customwidth) =>
+    <div className={`flex gap-5 items-center justify-center ${customwidth} ml-2`}>
+      <img alt="cursor" src={`/assets/arrowicon.svg`} className="cursor-pointer mt-3" />
+      <FormRenderer.FieldRenderer field={field} labelClass fieldWrapperClass />
+    </div>
+
+  let renderFields = () =>
+    switch selectedOption {
+    | "Greater than Equal to" => renderCommonFields(startamountField, "w-4/5")
+    | "Equal to" => <CustomAmountField customwidth="w-[162px]" />
```

Comment:
Please use rem instead of px , if some specified px doesn't have a rem mapping please add in tailiwind.config